runtime/browser/xwalk_runner.cc

```diff
@@ -144,6 +144,8 @@ void XWalkRunner::OnRenderProcessWillLaunch(content::RenderProcessHost* host) {
   std::vector<extensions::XWalkExtension*> ui_thread_extensions;
   std::vector<extensions::XWalkExtension*> extension_thread_extensions;
 
+// Disable bundled extensions for Linux desktop environment.
```

Comment:
```#if defined(OS_TIZEN) || !defined(OS_LINUX)``` is to enable code for all platforms other than Linux desktop.
Anyway I will change like @pozdnyakov mentioned.